scripts/bee.sh

```diff
@@ -21,6 +21,7 @@ PARAMETERS:
     --own-image                 If passed, the used Docker image names will be identical as the name of the workers.
     --version=x.y.z             used version of Bee client.
     --detach                    It will not log the output of Queen node at the end of the process.
+    --payment-tolerance         excess debt above payment threshold in BZZ where you disconnect from your peer (without decimals, default: 50000000000000)
```

Comment:
BTW I think we want to be able to set the `payment-threshold` and not necessarily `payment-tolerance`.

- the `payment-treshold` indicates when the cheques are sent https://docs.ethswarm.org/docs/installation/configuration#--payment-threshold
- the `payment-tolerance` as you already written down indicates when the peers disconnect unless they have cheques for the balance discrepancies. You can calculate it as `payment-tolerance < cumulative_sent_cheques_amount[peer] - cumulative_received_cheques_amount[peer] -balance[peer]` https://docs.ethswarm.org/docs/installation/configuration#--payment-tolerance

Reply:
you are right, I've changed to `payment-treshold`, somehow I swapped in my mind. thanks!